cmd/state-tray/main.go

```diff
@@ -17,13 +25,30 @@ func main() {
 func onReady() {
 	err := run()
 	if err != nil {
-		logging.Error("Systray encountered an error: %v", err)
+		logging.Error("Systray encountered an error: %v", errs.Join(err, ": "))
 		os.Exit(1)
 	}
 }
 
 func run() error {
-	logging.CurrentHandler().SetVerbose(true)
+	if os.Getenv("VERBOSE") == "true" {
+		// Doesn't seem to work, I think the systray lib and its logging solution is interfering
+		logging.CurrentHandler().SetVerbose(true)
+	}
+
+	stateSvcExe := filepath.Join(filepath.Dir(os.Args[0]), "state-svc")
+	if runtime.GOOS == "windows" {
+		stateSvcExe = stateSvcExe + ".exe"
+	}
+	if !fileutils.FileExists(stateSvcExe) {
+		return errs.New("Could not find: %s", stateSvcExe)
+	}
+
+	cmd := exec.Command(stateSvcExe, "start")
+	cmd.SysProcAttr = &syscall.SysProcAttr{HideWindow: true}
```

Comment:
nit/info:
Might be worth not making the Window at all: https://stackoverflow.com/a/59336873

Also interesting, from the same SO question: https://stackoverflow.com/a/46233782 - Output issues (i.e. VERBOSE comment above) may be related to this.

Reply:
Nice find!